Mathlib/RingTheory/Flat/Localization.lean

```diff
@@ -58,7 +58,8 @@ private lemma aux (I : Ideal R) (s : Submonoid R) :
   refine linearMap_ext s (mkLinearMap s _) (mkLinearMap s _) ?_
   ext m i
   rw [AlgebraTensorModule.curry_apply, curry_apply, restrictScalars_apply, LinearMap.comp_apply,
-    restrictScalars_apply, mkLinearMap_apply]
+    restrictScalars_apply, mkLinearMap_apply, AlgebraTensorModule.curry_apply, curry_apply,
+    restrictScalars_apply]
```

Comment:
Somehow we must get rid of the `restrictScalars` now, otherwise `simp` times out.